app/dcr_api/biz/service/job_service.go

```diff
@@ -146,3 +211,70 @@ func (js *JobService) GetJobAttestationReport(req *job.QueryJobAttestationReques
 	}
 	return j.AttestationReport, nil
 }
+
+func (js *JobService) getJobOutputFilename(UUID string, originName string) string {
+	return fmt.Sprintf("out-%s-%s", UUID, originName)
+}
+
+func (js *JobService) getJobOutputPath(creator string, UUID string, originName string) string {
+	return fmt.Sprintf("%s/output/%s", creator, js.getJobOutputFilename(UUID, originName))
+}
+
+func (g *JobService) getFileSize(remotePath string) (int64, error) {
```

Comment:
Will you change it to use an interface? Since we may support multiple cloud providers not only GCP

Reply:
good point. I believe this function would go away with #23, so I will keep it as is for now.